tools/libfrencutils/create_xgrid.c

```diff
@@ -212,19 +213,19 @@ int create_xgrid_1dx2d_order2(const int *nlon_in, const int *nlat_in, const int
       lon_in_avg = avgval_double(n_in, x_in);
 
       if (  (n_out = clip ( x_in, y_in, n_in, ll_lon, ll_lat, ur_lon, ur_lat, x_out, y_out )) > 0 ) {
-	xarea = poly_area (x_out, y_out, n_out ) * mask_in[j1*nx1+i1];
+  xarea = poly_area (x_out, y_out, n_out ) * mask_in[j1*nx1+i1];
         min_area = min(area_in[j1*nx1+i1], area_out[j2*nx2+i2]);
-	if(xarea/min_area > AREA_RATIO_THRESH ) {
-	  xgrid_area[nxgrid] = xarea;
-	  xgrid_clon[nxgrid] = poly_ctrlon(x_out, y_out, n_out, lon_in_avg);
-	  xgrid_clat[nxgrid] = poly_ctrlat (x_out, y_out, n_out );
-	  i_in[nxgrid]    = i1;
-	  j_in[nxgrid]    = j1;
-	  i_out[nxgrid]   = i2;
-	  j_out[nxgrid]   = j2;
-	  ++nxgrid;
-	  if(nxgrid > MAXXGRID) error_handler("nxgrid is greater than MAXXGRID, increase MAXXGRID");
-	}
+  if(xarea/min_area > AREA_RATIO_THRESH ) {
```

Comment:
Which editor or IDE are you using and is it it able to use the file FRE-NCTools/.editorconfig ? (PS Emacs and VSCode should be able to use it) I am not sure, but either :
a) your editor may be breaking the indentation of some of these lines. 
b) we may have to upgrade file .editorconfig
c) This file (and a few others) was never formatted post the intro of .editorconfig in the project. 
PS If you do decide to apply whole file reformatting, we may not be able to do so for the "usage" comment near 
the top.
PPS This also applies to other files (or functions) that were worked on.

Reply:
Why am I having these spacing issues all of a sudden?  Is this .editorconfig file new?

Reply:
I did download the editorconfig plugin for emacs but these mysterious differences are still showing up

Reply:
@mlee03 
OK I'll accept it. Relating to this we've had a historical and informal modus operandi you might want to be aware of. With the intent of helping the reviewer, usually we have not reformatted entire files, but only those functions that were needed to be modified to fix the bug or add the enhancement. This ultimately stemmed from the fact that we have limited unit tests, and so reviewers must look carefully at those lines that are truly modified or changed; looking at other lines is extra work  and effort that detracts from checking correctness.  If it was desirable to reformat entire files just to upgrade the format, this was done in a separate push and possibly with a different PR.